tests/shared/src/test/scala/cats/tests/ApplicativeSuite.scala

```diff
@@ -39,6 +41,18 @@ class ApplicativeSuite extends CatsSuite {
     assert(fa.replicateA(5) === (Some(List(1, 1, 1, 1, 1))))
   }
 
+  test("replicateA_ executes the Applicative action 'fa' 'n' times") {
+    val A = Applicative[Option]
+    val fa = A.pure(0)
+    assert(fa.replicateA_(5) === Option(unit))
```

Comment:
Just a minor bit: can we exercise values other than 5?

Actually, can you make a loop and try all values from 0 up to 10 or something?